packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -231,7 +231,20 @@ const main = async () => {
             new program.Option('--inspect', 'enable debugging with --inspect on the node process')
         )
         .addOption(new program.Option('--noHMR', 'disable the client-side hot module replacement'))
-        .action(async ({inspect, noHMR}) => {
+        .addOption(
+            new program.Option(
+                '--babelArgs <babel-args>',
+                'args to pass through to babel'
+            ).argParser((val) => {
+                if (typeof val !== 'string' || val === '') {
+                    throw new program.InvalidArgumentError(`"babel-args" cannot be empty`)
+                } else {
+                    return val
+                }
+            })
```

Comment:
```suggestion
            )
```
`--babel-args` is an optional flag.

Reply:
If a user provides `babelArgs` that does not include `--extensions` (e.g. `--ignore some-package`), do we want to add the missing `--extensions` so that the command continues to work? Or do we want `--extensions` to simply be a default value, and it becomes the responsibility of the user if they replace it?

I think the latter is a reasonable stance for the first iteration of this flag, we'd implement with this:
```suggestion
            ).default('--extensions ".js,.jsx,.ts,.tsx"')
```
Along with removing the default param on L246:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/403ce85d34062e5de1d36a1dff1520dcbe95161b/packages/pwa-kit-dev/bin/pwa-kit-dev.js#L246